cmd/gpu-kubelet-plugin/main.go

```diff
@@ -147,6 +148,13 @@ func newApp() *cli.App {
 			Destination: &flags.healthcheckPort,
 			EnvVars:     []string{"HEALTHCHECK_PORT"},
 		},
+		&cli.StringFlag{
+			Name:        "additional-xids-to-ignore",
+			Usage:       "A comma-separated list of additional XIDs to ignore.",
+			Value:       "",
+			Destination: &flags.additionalXidsToIgnore,
+			EnvVars:     []string{"ADDITIONAL_XIDs_TO_IGNORE"},
+		},
```

Comment:
In https://github.com/NVIDIA/k8s-device-plugin/pull/1443 we added a list of EXPLICIT XIDs to consider fatal. This allows a user to:
1. Specify ignored XIDs (including `all`)
2. Specify SPECIFIC XIDs that are considered fatal (including `all`).

The important thing here is that it allows users to  override the list of hard-coded XIDs that we currently track.